### DISCUSSION TITLE
Autocomplete in VSCode

### DISCUSSION THREAD

**Discussion by Joebayld** [vuetifyjs/vuetify#17915] [Q&A]
Is it possible to get autocomplete working in VSCode when typing out component names? 

Would like to type `VBu...` and it suggests VButton etc.

**Comment by yuwu9145** [comment]
With JSX + typescript, you can 

![image](https://github.com/vuetifyjs/vuetify/assets/5698884/2912e479-1511-493d-b771-94ccaf66f280)
